src/components/TreesMap/DeckGLMap.tsx

```diff
@@ -600,65 +604,54 @@ class DeckGLMap extends React.Component<DeckGLPropType, DeckGLStateType> {
+                      coords: {
+                        latitude: number;
+                        longitude: number;
                       };
-                      this.setViewport({
-                        longitude,
-                        latitude,
-                        zoom: VIEWSTATE_ZOOMEDIN_ZOOM,
-                        transitionDuration: VIEWSTATE_TRANSITION_DURATION,
-                      });
-                    }}
-                  />
-                  <NavigationControl
-                    onViewStateChange={e =>
-                      this.setViewport({
-                        latitude: e.viewState.latitude,
-                        longitude: e.viewState.longitude,
-                        zoom: e.viewState.zoom,
-                        transitionDuration: VIEWSTATE_TRANSITION_DURATION,
-                      })
-                    }
-                  />
-                </ControlWrapper>
-                {!isMobile &&
-                  hoveredPump &&
-                  hoveredPump.latitude &&
-                  hoveredPump.longitude && (
-                    <Popup
-                      latitude={hoveredPump.latitude}
-                      longitude={hoveredPump.longitude}
-                      closeButton={false}
-                      closeOnClick={false}
-                      sortByDepth={true}
-                    >
-                      <DataTable
-                        title='Öffentliche Straßenpumpe'
-                        subtitle={hoveredPump.address}
-                        items={{
-                          Status: hoveredPump.status,
-                          'Letzter Check': hoveredPump.check_date,
-                          Pumpenstil: hoveredPump.style,
-                        }}
-                      />
-                    </Popup>
-                  )}
-              </>
+                    };
+                    this.setViewport({
+                      longitude,
+                      latitude,
+                      zoom: VIEWSTATE_ZOOMEDIN_ZOOM,
+                      transitionDuration: VIEWSTATE_TRANSITION_DURATION,
+                    });
+                  }}
+                />
+                <NavigationControl
+                  onViewStateChange={e =>
+                    this.setViewport({
+                      latitude: e.viewState.latitude,
+                      longitude: e.viewState.longitude,
+                      zoom: e.viewState.zoom,
+                      transitionDuration: VIEWSTATE_TRANSITION_DURATION,
+                    })
+                  }
+                />
+              </ControlWrapper>
             )}
           </StaticMap>
+          {hoveredPump && hoveredPump.x && hoveredPump.y && (
+            <Tooltip
+              x={hoveredPump.x}
+              y={hoveredPump.y}
+              title='Öffentliche Straßenpumpe'
+              subtitle={hoveredPump.address}
+              infos={{
+                Status: hoveredPump.status,
```

Comment:
This element is called Tooltip but it seems to be specific to Pumps. Either make it generic or call it something specific like `MapPumpTooltip`.

Reply:
My bad! I do understand now that each key is one column of the table and each value is the second column. Greate Idea! I would, however, still choose a more specific name for the Component, even if it is located in the `TreesMap` folder. :)

Reply:
You're right, it's a `MapTooltip`, see commit 3072210 for the change.

And standing up for my pumps now: Is the `TreesMap` not actually a `TreesAndPumpsMap`? ;) Kidding. But actually there could be a broader term for it?!